src/ssh/azext_ssh/_params.py

```diff
@@ -20,6 +20,7 @@ def load_arguments(self, _):
         c.argument('cert_file', options_list=['--certificate-file', '-c'],
                    help='Path to a certificate file used for authentication when using local user credentials.')
         c.argument('port', options_list=['--port'], help='SSH port')
+        c.argument('resource_tag', options_list=['--resource-tag'], help='Use port value from specified resource tag.')
```

Comment:
I don't know if the message is defined in the document that Danny wrote, but feel free to ask Danny for his suggestions on customer facing messages like this (as well as error and warning messages). Also, we should add examples on how to use this tag in the help as well, we can chat with Danny to decide what we want to add.

Reply:
Just sent him a message about it